AlphaWallet/ActiveWalletCoordinator.swift

```diff
@@ -28,11 +28,13 @@ class ActiveWalletCoordinator: NSObject, Coordinator {
     private let activitiesPipeLine: ActivitiesPipeLine
     private let sessionsProvider: SessionsProvider
     private let currencyService: CurrencyService
+    private let tokenGroupIdentifier: TokenGroupIdentifierProtocol
```

Comment:
Not this PR, but missed this one a while back. But better to rename `TokenGroupIdentifierProtocol` to `TokenGroupMakerProtocol`. The "identifier" suggests IDs. Ambiguous. And `identifier(fromFileName:)` too. `identify(token:)` isn't so bad.

Reply:
Maybe just `TokenGroupProtocol`? We are not making any groups, just identifying the groups.